src/components/Marketplace/Catalog/FeaturedToken.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { Token } from '../../../reducer/slices/collections';
+import { useLocation } from 'wouter';
+import { ipfsUriToGatewayUrl } from '../../../lib/util/ipfs';
+import { AspectRatio, Box, Container, Flex, Spacer, Text, Heading, Stack } from '@chakra-ui/react';
+import { MinterButton } from '../../common';
+import { TokenMedia } from '../../common/TokenMedia';
+
+interface FeaturedTokenProps extends Token {
+  network: string;
+}
+
+export default function FeaturedToken(props: FeaturedTokenProps) {
```

Comment:
This breaks with small width:

![image](https://user-images.githubusercontent.com/703880/112415260-8b7a7f00-8cf1-11eb-90de-9e8bb4013f20.png)

Reply:
> This breaks with small width:
> 
> ![image](https://user-images.githubusercontent.com/703880/112415260-8b7a7f00-8cf1-11eb-90de-9e8bb4013f20.png)

Can you tell me what browser version and OS you are using? It is coded that the image and content stack on top of each other in small screens. Possible Chakra-ui bug here.